config/routes.rb

```diff
@@ -2,6 +2,8 @@
   root 'flights#index'
   get 'flights' => 'flights#index'
   get 'bookings' => 'bookings#new'
+  get 'new_bookings' => 'bookings#create'
+  get 'show_bookings' => 'bookings#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.